app/src/main/java/com/example/sopt_seminar/ui/FollowerFragment.kt

```diff
@@ -25,49 +22,22 @@ class FollowerFragment : BaseFragment<FollowerFragmentBinding>(R.layout.follower
             findNavController().navigate(action)
         }
 
-        val simpleCallback = object : ItemTouchHelper.SimpleCallback(
-            ItemTouchHelper.UP or ItemTouchHelper.DOWN,
-            ItemTouchHelper.LEFT
-        ) {
-            override fun onMove(
-                recyclerView: RecyclerView,
-                viewHolder: RecyclerView.ViewHolder,
-                target: RecyclerView.ViewHolder
-            ): Boolean {
-                val startPosition = viewHolder.adapterPosition
-                val endPosition = target.adapterPosition
-                adapter.moveItem(startPosition, endPosition) {
-                    moveItem(startPosition, endPosition)
-                }
-                return true
-            }
-
-            override fun onSwiped(viewHolder: RecyclerView.ViewHolder, direction: Int) {
-                when (direction) {
-                    ItemTouchHelper.LEFT -> {
-                        adapter.removeItem(viewHolder.adapterPosition) {
-                            removeItem(viewHolder.adapterPosition)
-                        }
-                    }
-                }
-            }
-        }
-
-        with(binding) {
+        binding.apply {
+            main = this@FollowerFragment
             followerRecyclerView.adapter = adapter
-            ItemTouchHelper(simpleCallback).attachToRecyclerView(followerRecyclerView)
-            adapter.submitList(testList.toCollection(mutableListOf()))
         }
     }
 
-    private fun removeItem(position: Int) {
+    val removeItem = fun(position: Int) {
         testList.removeAt(position)
     }
```

Comment:
뭔가 함수를 변수화 시킬거면 람다식으로 아래처럼 좀 더 간결하게 표현 가능할듯? 참고바람!

```suggestion
    val removeItem : ((Int) -> Unit) = { position ->
        testList.removeAt(position)
    }
```